modules/unsupported/csv/src/test/java/org/geotools/data/csv/CSVWriteTest.java

```diff
@@ -250,11 +250,20 @@ public void replaceAll() throws Exception {
         SimpleFeatureStore featureStore = (SimpleFeatureStore) store.getFeatureSource("locations");
         assertEquals("featureStore should only have the one feature we created", 
                 1, featureStore.getFeatures().size());
-        String contents = "LAT,LON,CITY,NUMBER,YEAR\nPOINT (45.52 -122.681944),Portland,800,2014,";
+        String contents;
+        if (isWindows() == true) {
+        	contents = "LAT,LON,CITY,NUMBER,YEAR\r\nPOINT (45.52 -122.681944),Portland,800,2014,";
+        } else {
+        	contents = "LAT,LON,CITY,NUMBER,YEAR\nPOINT (45.52 -122.681944),Portland,800,2014,";
+        }
```

Comment:
Use the following rather than isWindows()

````
String contents = "LAT,LON,CITY,NUMBER,YEAR"+System.getProperty("line.separator")"+"POINT (45.52 -122.681944),Portland,800,2014,";
```